### PR TITLE
Fix BuildFiles, includes and comments

### PR DESCRIPTION
**Pull request by fwyzard**
#### PR description:

Fix BuildFiles, includes and comments.

No changes expected.

#### PR validation:

None.

### CODE REVIEW

File: DataFormats/Portable/BuildFile.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <use name="alpaka"/>
 <use name="rootcling"/>
 <use name="FWCore/Utilities/" source_only="1"/>
-<use name="HeterogeneousCore/AlpakaInterface" source_only="1"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
```
Comment (fwyzard) [inline DataFormats/Portable/BuildFile.xml:4]: This because `source_only="1"` does not propagate the module's external dependencies.

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
+heterogeneous

**Comment by jfernan2** [comment]
+1

**Comment by mandrenguyen** [comment]
+1

**Pull request merged**